osxcollector/output_filters/virustotal/lookup_urls.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+#
+# LookupURLsFilter uses VirusTotal to lookup the URLs in 'LSQuarantineDataURLString' and add 'osxcollector_vturl' key.
+#
+import re
+
+from osxcollector.output_filters.base_filters.output_filter import run_filter
+from osxcollector.output_filters.base_filters. \
+    threat_feed import ThreatFeedFilter
+from osxcollector.output_filters.virustotal.api import VirusTotalApi
+
+
+class LookupURLsFilter(ThreatFeedFilter):
+
+    """A class to find suspicious URLs using VirusTotal API."""
+
+    SCHEMES = re.compile('https?')
+
+    def __init__(self, only_lookup_when=None, is_suspicious_when=None):
+        lookup_when_url_scheme_matches = self._generate_only_lookup_when(only_lookup_when)
+        super(LookupURLsFilter, self).__init__('LSQuarantineDataURLString', 'osxcollector_vturl',
```

Comment:
The signature of ThreatFeedFilter has changed to 
```python
def __init__(self, ioc_key, output_key, lookup_when=None, name_of_api_key=None):
```

You will need to rename `only_lookup_when` and can get rid of `is_suspicious_when`.